lib/rules/one-var.js

```diff
@@ -53,6 +84,34 @@ module.exports = function(context) {
         }
     }
 
+    /**
+     * Determines if there is more than one let statement in the current scope.
+     * @returns {boolean} Returns true if it is the first let declaration, false if not.
```

Comment:
Sorry, I commented without seeing your second comment.

Yes. I just just poorly named, the logic works.